README.md

```diff
@@ -1,4 +1,371 @@
-# configuration
+
+# Contributing
```

Comment:
Move this part to other part of this doc. I want to know what this project is when I open this doc.

Reply:
moved